src/components/Button.tsx

```diff
@@ -30,23 +33,37 @@ type Props = {
     children: React.ReactElement | string;
     priority?: 'primary' | 'secondary';
     showArrow?: boolean;
+    isTertiary?: boolean;
```

Comment:
Well... I tried that but couldn't make it work. I agree it would be much better to standardise the component interface and treat `tertiary` as a `priority` not a random flag, but TS wasn't happy with the idea of having `tertiary` there as not all components downstream support that.